Sources/TuistKit/Utils/CommentsRemover.swift

```diff
@@ -0,0 +1,19 @@
+import Foundation
+
+enum CommentsRemover {
+    static func removeComments(from code: String) -> String {
+        let regexPattern = #"//.*?$|/\*[\s\S]*?\*/"#
+        do {
+            let regex = try NSRegularExpression(pattern: regexPattern, options: [.anchorsMatchLines])
+            let range = NSRange(location: 0, length: code.utf16.count)
+            var cleanCode = regex.stringByReplacingMatches(in: code, options: [], range: range, withTemplate: "")
+
+            cleanCode = cleanCode.replacingOccurrences(of: #"(?m)^\s*\n"#, with: "", options: .regularExpression)
+
+            return cleanCode
+        } catch {
+            print("Invalid regex pattern: \(error)")
```

Comment:
for errors we can use logger instead of print statements - that said do we expect this to error at runtime based on the input? perhaps make the entire method throwing and bubble up the error?